bazel/ray.bzl

```diff
@@ -140,3 +140,55 @@ def copy_to_workspace(name, srcs, dstdir = ""):
         ),
         local = 1,
     )
+
+def native_java_binary(module_name, name, native_binary_name):
+    native.genrule(
+        name = name + "_darwin",
+        srcs = [native_binary_name],
+        outs = [module_name + "/src/main/resources/native/darwin/" + name],
+        cmd = "cp $< $@",
+        output_to_bindir = 1,
+    )
+
+    native.genrule(
```

Comment:
Is it possible to use `copy_file` from `@bazel_skylib//rules:copy_file.bzl` instead of these `genrule`s? If so, please do. (Though it might not be; if not then never mind.)

Reply:
thanks, `copy_file` works.